test/simulation/LICENSE

```diff
@@ -0,0 +1,661 @@
+                    GNU AFFERO GENERAL PUBLIC LICENSE
```

Comment:
If it's in the same repo, I believe this will be covered by the license in the root folder without needing to reproduce it here.